src/Commands/StartCommand.php

```diff
@@ -16,7 +16,7 @@ class StartCommand extends Command implements SignalableCommandInterface
     public $signature = 'octane:start
                     {--server= : The server that should be used to serve the application}
                     {--host=127.0.0.1 : The IP address the server should bind to}
-                    {--port=8000 : The port the server should be available on}
```

Comment:
@hendrikheil  This change is safe but slightly inconvenient as it removes the default port from the `--help` text of the command:
```
// Before
--port[=PORT]                  The port the server should be available on [default: "8000"]

// After
--port[=PORT]                  The port the server should be available on
```

Might be worth to add this to the text :+1:

Reply:
@Namoshek good suggestion. Added that: https://github.com/laravel/octane/blob/1.x/src/Commands/StartCommand.php